docs/source/logging/logging.md

```diff
@@ -45,4 +45,71 @@ To start this example execute:
 RUST_LOG=info cargo run
 ```
 
-The full [example](https://github.com/scylladb/scylla-rust-driver/tree/main/examples/logging.rs) is available in the `examples` folder
\ No newline at end of file
+The full [example](https://github.com/scylladb/scylla-rust-driver/tree/main/examples/logging.rs) is available in the `examples` folder
```

Comment:
Quite a side topic, but I couldn't resist investigating it.

This usage of *folder* instead of the usual Unixy *directory* made me wonder about the very difference between these two terms, and these are some opinions:
- https://www.baeldung.com/cs/directories-vs-folders -> folder is a "leaf directory", i.e. directory without directories in it;
- https://superuser.com/questions/187900/what-is-the-difference-between-a-directory-and-a-folder -> no strong difference; *directory* more of a file system concept, *folder* more of a GUI thing.